plugins/file/index.js

```diff
@@ -141,12 +144,7 @@ module.exports = {
           params.CacheControl = options.cacheControl;
         }
 
-        const data = await upload(s3, params);
-
-        return {
-          key: data.Key,
-          url: data.Location,
-        };
+        return upload(s3Client, params, options.s3Region);
```

Comment:
```suggestion
        return await upload(s3Client, params, options.s3Region);
```

Oh sorry, I just saw this. Doesn't this have to have an await?- Because upload is an async function?